xlators/cluster/dht/src/dht-selfheal.c

```diff
@@ -2162,6 +2171,8 @@ dht_dir_heal_xattrs(void *data)
                         DHT_MSG_DIR_XATTR_HEAL_FAILED,
                         "set-user-xattr-failed path=%s", local->loc.path,
                         "subvol=%s", subvol->name, "gfid=%s", gfid, NULL);
+            } else {
+                dict_del(xdata, "sync_backend_xattrs");
```

Comment:
```suggestion
                dict_del_sizen(xdata, "sync_backend_xattrs");
```

Reply:
The change does not exist in devel branch.First i have to merge the changes in devel branch so i will send a separate patch to handle this.